packages/blade/CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # @razorpay/blade
 
+## 2.0.0
+
+### Major Changes
+
+- cc4355a: feat(blade): added 2px spacing token
+
+  ## Breaking changes
+
+  - Added 2px token, thus all spacing tokens have shifted by 1 step.
+
+  ## Migration steps
+
+  Shift every spacing token other than the first one (`0th` index) by +1
```

Comment:
We should mention if people not using the tokens directly need to change anything or not

Reply:
That's i think implicit anyways. If no tokens are being used no changes needed.

Reply:
Wasn't very evident to me :)